articles/azure-app-configuration/concept-geo-replication.md

```diff
@@ -44,6 +44,12 @@ To add once these links become available:
 
 Each replica created will add extra charges. Reference the [App Configuration pricing page](https://azure.microsoft.com/pricing/details/app-configuration/) for details. As an example, if your origin is a standard tier configuration store and you have five replicas, you would be charged the rate of six standard tier configuration stores for your system, but each of your replica's isolated quota and requests are included in this charge.
 
+## Monitoring
+
+In Azure App Configuration, effective monitoring involves tracking replication latency, which measures the time it takes for changes in one region to sync with replicas in other regions. 
```

Comment:
> effective monitoring involves tracking replication latency

I feel that's a pretty strong statement. I know there are some users who are interested in monitoring and alerting on replication latency. But I think an even bigger value of it will be for users to get a sense of comfortability with the system. And perhaps after an initial discovery stage they may no longer opt to monitor the metric. The way it's currently worded makes it seem that the expectation is to have an alert on replication latency.